### PR TITLE
rewrite readme to point towards initiatives

### PR DESCRIPTION
**Pull request by oe**
This should give outside users a good perspective on the work we do here vs. the work done on our initiatives, and hopefully point them in the right direction.

### CODE REVIEW

**Comment by chowdhurian** [comment]
Maybe:

- Community Committee (meta-level concerns, cross-cutting with other teams)
  - Initiatives (focused on specific tasks, independent from the Community Committee). For example, the Internationalization Initiative is focused on translation of various Node.js resources.

This also combines the example sub-item into the first one. Just a suggestion for clarity.

**Comment by oe** [comment]
Would love to get a final round of feedback from @nodejs/community-committee!

File: README.md [hunk 1]
```diff
@@ -11,7 +11,14 @@ The Community Committee is tasked with further building out the Node.js Communit
 
 As a part of the Node.js Community, we'd love to have your help! To **get started** with contributing, you can check out the Issues in this repository labeled "Good First Issue" to see where we're looking for help. If you have your own ideas on how we could engage and build the community, feel free to create your own issue!
 
-If you're interested in participating in the Community Committee as a member, you should create an issue asking to be an Observer in our next Community Committee meeting. You can find a great example of such an issue [here](https://github.com/nodejs/community-committee/issues/142)!
+As a part of the Node.js Community, we'd love to have your help! Before you get started, here's a broad outline of the Community Committee's governance structure:
+
+- Community Commitee (meta-level concerns, cross-cutting with other teams)
+  - Initiatives (focused on specific tasks, independent from the Community Committee). For example, the Internationalization Initiative, which is focused on the translation of various Node.js resources.
+
```
Comment (mhdawson) [inline README.md]: W

I think that Internationalization was chartered as a team:

we might want to include the overview/references to:

Strategic Initiatives
Working Groups (ex Internationalization)
Teams

Comment (oe) [reply README.md]: @mhdawson Working Groups can't be chartered as we haven't passed any governance rules regarding that. i18n, as far as I know, has been mistakenly created as a Working Group. But yeah, I'll update that with another example.

Comment (bnb) [reply README.md]: @mhdawson Yeah, this PR made me realize that we don't have official governance around WGs as @oe pointed out. I _believe_ Evangelism can be grandfathered in, but there's not good basis for that belief.

FWIW @dshaw begun work on taking this on back in December: https://github.com/nodejs/community-committee/issues/201

**Comment by oe** [comment]
Merging this!

**Pull request merged**